examples/library/test_get_cycle.cpp

```diff
@@ -55,12 +55,8 @@ int test_get_cycle (int argc, char **argv) {
         config = hb_mc_manycore_get_config(&mc);
 
         // To cause time to proceed, we write to DRAM.
-        uint32_t manycore_dim_x = hb_mc_coordinate_get_x(hb_mc_config_get_dimension_vcore(config));
-        uint32_t manycore_dim_y = hb_mc_coordinate_get_y(hb_mc_config_get_dimension_vcore(config));
-
-        uint32_t dram_coord_x = 0;
-        uint32_t dram_coord_y = hb_mc_config_get_dram_y(config);
-        hb_mc_npa_t npa = { .x = dram_coord_x, .y = dram_coord_y, .epa = 0};
+        hb_mc_coordinate_t dram_coord = hb_mc_config_get_dram_coordinate(config, 0);
+        hb_mc_npa_t npa = hb_mc_npa(dram_coord, 0);
```

Comment:
Good. I think we have an EPA DRAM, too. I thought I added it but it might be in my network simulator PR

Reply:
https://github.com/bespoke-silicon-group/bsg_replicant/blob/2af7bfecabacd28349ec3362dbb387a8d39dab78/libraries/bsg_manycore_vcache.h#L50